client/landing/gutenboarding/onboarding-block/design-selector/index.tsx

```diff
@@ -23,20 +26,23 @@ export default () => {
 
 	const homepageTemplates = templates.filter( template => template.category === 'home' );
 
-	const [ previewedTemplate, setPreviewedTemplate ] = useState< string | null >( null );
+	const [ selectedDesign, setSelectedDesign ] = useState< string | undefined >();
 	return (
-		<div
-			className="page-template-modal__list" // eslint-disable-line wpcalypso/jsx-classname-namespace
-		>
-			<TemplateSelectorControl
-				label={ NO__( 'Layout', 'full-site-editing' ) }
-				templates={ homepageTemplates }
-				blocksByTemplates={ {} /* Unneeded, since we're setting `useDynamicPreview` to `false` */ }
-				onTemplateSelect={ setPreviewedTemplate }
-				useDynamicPreview={ false }
-				siteInformation={ undefined }
-				selectedTemplate={ previewedTemplate }
-			/>
+		<div className="design-selector">
+			{ homepageTemplates.map( t => (
```

Comment:
Maybe call the var `template`, to make it more self-explanatory in case the whole things grows more.

```suggestion
			{ homepageTemplates.map( template => (
```